src/css/tabs/servos.css

```diff
@@ -188,3 +188,306 @@
         min-width: 60px;
     }
 }
+
+/* from Motors TAB */
+
+.tab-servos .ui-grid-col {
+    margin-bottom: 0;
+}
+/*
+.tab-servos input,
+*/
+.tab-servos select {
+    border: 1px solid var(--subtleAccent);
+    width: 230px;
+    height: 20px;
+    float: left;
+    margin-right: 15px;
+    border-radius: 3px;
+}
+
+/*
+.tab-servos table {
+    margin-bottom: 0;
+    width: 100%;
+    float: left;
+}
+*/
+.tab-servos table, .tab-servos table th, .tab-servos table td {
+    padding: 0;
+    text-align: left;
+}
+
+.tab-servos table th {
+    padding: 3px;
+    border-bottom: 1px solid var(--subtleAccent);
+}
+
+.tab-servos table td {
+    padding: 5px 3px;
+    border-bottom: 1px solid var(--subtleAccent);
+}
+
+.tab-servos table tr td:first-child {
+    text-align: left;
+    width: 55px;
+}
+
+.tab-servos table thead tr:first-child {
+    background-color: #ececec;
+}
+
+.tab-servos {
+    position: relative;
+}
+
+.tab-servos .groupTitle {
+    padding: 0 0 5px 0;
+    margin: 0 0 10px 0;
+    font-size: 16px;
+    border-bottom: 1px solid var(--subtleAccent);
+}
+
+.tab-servos .number input {
+    width: 55px;
+    padding-left: 3px;
+    height: 20px;
+    line-height: 20px;
+    text-align: left;
+    border-radius: 3px;
+    margin-right: 11px;
+    font-size: 12px;
+    font-weight: normal;
+}
+
+.tab-servos .number .disabled {
+    width: 48px;
+    padding: 0 5px;
+    background-color: #ececec;
+}
+
+.tab-servos .number span {
+    margin-left: 0;
+}
+
+.tab-servos .freelabel {
+    margin-left: 10px;
+    position: relative;
+}
+
+.tab-servos span {
+    margin: 0;
+}
+
+.tab-servos .spacer_box {
+    padding-bottom: 10px;
+    float: left;
+    width: calc(100% - 20px);
+}
+
+.tab-servos .number,
+.tab-servos .select
+{
+    margin-bottom: 5px;
+    clear: left;
+    padding-bottom: 5px;
+    border-bottom: 1px solid var(--subtleAccent);
+    width: 100%;
+    float: left;
+}
+
+.tab-servos .number:last-child,
+.tab-servos .select:last-child {
+    border-bottom: none;
+    padding-bottom: 0;
+    margin-bottom: 0;
+}
+
+.tab-servos .gui_box_titlebar {
+    margin-bottom: 0;
+}
+
+.tab-servos .numberspacer {
+    float: left;
+    width: 65px;
+    height: 21px;
+}
+
+.tab-servos thead {
+    display: none;
+}
+
+.tab-servos .gui_box {
+    margin-bottom: 10px;
+    float: left;
+    font-weight: bold;
+}
+
+.tab-servos .gui_box span {
+    font-style: normal;
+    font-weight: normal;
+    line-height: 19px;
+    color: #4f4f4f;
+    font-size: 11px;
+}
+
+.tab-servos .alignicon {
+    width: 15px;
+    height: 15px;
+    float: left;
+    margin: 3px;
+}
+
+.tab-servos .spacer {
+    width: calc(100% - 34px);
+    margin: 10px;
+}
+
+.tab-servos .servoblock {
+    margin-bottom: 0;
+    background-color: #ECECEC;
+    /*background-image: -webkit-linear-gradient(top, transparent, rgba(0, 0, 0, 0.15));*/
```

Comment:
If we don't use this then it should be removed.

Reply:
Removed